go.mod

```diff
@@ -12,132 +12,148 @@ require (
 	github.com/docker/go-units v0.5.0
 	github.com/dop251/goja v0.0.0-20230427124612-428fc442ff5f
 	github.com/dubbogo/gost v1.12.3
-	github.com/gin-gonic/gin v1.8.1
-	github.com/go-playground/validator/v10 v10.11.1
+	github.com/gin-gonic/gin v1.9.1
+	github.com/go-playground/validator/v10 v10.14.0
 	github.com/go-sql-driver/mysql v1.6.0
 	github.com/golang/mock v1.6.0
 	github.com/google/btree v1.1.2
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.4.0
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/nacos-group/nacos-sdk-go/v2 v2.0.1
 	github.com/natefinch/lumberjack v2.0.0+incompatible
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.11.1
+	github.com/prometheus/client_golang v1.14.0
 	github.com/shopspring/decimal v1.3.1
 	github.com/spf13/cast v1.3.1
 	github.com/spf13/cobra v1.2.1
-	github.com/stretchr/testify v1.8.3
-	github.com/testcontainers/testcontainers-go v0.12.0
+	github.com/stretchr/testify v1.8.4
+	github.com/testcontainers/testcontainers-go v0.27.0
```

Comment:
Upgrade the `testcontainer-go` may cause the code compile error.

error file: `https://github.com/arana-db/arana/blob/master/test/testcontainer_mysql.go#L62`

ref: https://github.com/testcontainers/testcontainers-go/pull/386/files